src/VisualStudio/Core/Def/Implementation/FindReferences/FindUsagesUtilities.cs

```diff
@@ -0,0 +1,87 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Classification;
+using Microsoft.CodeAnalysis.Host;
+using Microsoft.CodeAnalysis.Text;
+using Roslyn.Utilities;
+
+namespace Microsoft.VisualStudio.LanguageServices.FindUsages
+{
+    internal static class FindUsagesUtilities
```

Comment:
This contains methods moved from `AbstractTableDataSourceFindUsagesContext.cs` and `AbstractDocumentSpanEntry.cs`